modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -346,7 +346,9 @@ public ObjectDefinition putObjectDefinition(
 				titleObjectFieldId,
 				GetterUtil.getBoolean(
 					objectDefinition.getAccountEntryRestricted()),
-				GetterUtil.getBoolean(objectDefinition.getActive(), true),
+				GetterUtil.getBoolean(
+					objectDefinition.getActive(),
+					serviceBuilderObjectDefinition.getActive()),
```

Comment:
@locpham97 @TaiPhamLR why did you add the current value of the `serviceBuilderObjectDefinition` here, but you didn't add it below in `GetterUtil.getBoolean(objectDefinition.getPortlet())`?

Reply:
Hi @carolmariaabb. Before changing these line codes (349-351), an exception was thrown at [ObjectDefinitionLocalServiceImpl#_validateActive](https://github.com/liferay-objects/liferay-portal/blob/9758d79ea9edcfa3be635da813bcc1aa54b74b9d/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java#L1337) if 'active' was not set in request json. I use the 'active' value of serviceBuilderObjectDefinition (it's an old value in the database) in case the input json does not include 'active'.
If I don't understand properly, please let me know.
Thanks a lot.